### PR TITLE
Prepare configuration for 2019.1+ release (0.18.x)

### PR DESCRIPTION
**Pull request by King2500**
Here is the update for gradle and travis configuration for 2019.1+ release.

Keep in mind to update `plugin.xml` to 0.18.x when you release.

### CODE REVIEW

**Comment by King2500** [comment]
Some of the failing tests also failed in #1257

**Comment by King2500** [comment]
@Haehnchen and maybe @cedricziel:

Any chance you can look after this ?

**Comment by cedricziel** [comment]
Soo, to move forward, i'll work on your changes, @King2500 . I think these are nice additions and I also think we can use some fresh air by removing support for anything < 2019.1.

**Comment by cedricziel** [comment]
Also, @Haehnchen seems to be onboard with a v0.18, so I'm fine as well :)

File: gradle.properties [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
+#ideaVersion = IU-191.6707.31-EAP-SNAPSHOT
+#phpPluginVersion = 191.6707.42
+#twigPluginVersion = 191.6183.95
 #toolboxPluginVersion = 0.4.6
 #annotationPluginVersion = 5.3
 
-ideaVersion = IU-2018.3.6
-phpPluginVersion = 183.5429.47
-twigPluginVersion = 183.3795.24
+ideaVersion = IU-2019.1
```
Comment (cedricziel) [inline gradle.properties]:
```suggestion
ideaVersion = IU-2019.1.1
```

File: .travis.yml [hunk 2]
```diff
@@ -22,10 +22,8 @@ before_install:
...
 
 env:
-  - PHPSTORM_ENV="skip incomplete" IDEA_VERSION="IU-183.2153.8" PHP_PLUGIN_VERSION="183.2153.44" TWIG_PLUGIN_VERSION="183.2153.44" TOOLBOX_PLUGIN_VERSION="0.4.6" ANNOTATION_PLUGIN_VERSION="5.3"
-  - PHPSTORM_ENV="skip incomplete" IDEA_VERSION="IU-2018.2.5" PHP_PLUGIN_VERSION="182.4892.16" TWIG_PLUGIN_VERSION="182.3458.35" TOOLBOX_PLUGIN_VERSION="0.4.6" ANNOTATION_PLUGIN_VERSION="5.3"
-  - PHPSTORM_ENV="skip incomplete" IDEA_VERSION="IU-2018.2" PHP_PLUGIN_VERSION="182.3684.42" TWIG_PLUGIN_VERSION="182.3458.35" TOOLBOX_PLUGIN_VERSION="0.4.6" ANNOTATION_PLUGIN_VERSION="5.3"
-  - PHPSTORM_ENV="skip incomplete" IDEA_VERSION="IU-2018.3.6" PHP_PLUGIN_VERSION="183.5429.47" TWIG_PLUGIN_VERSION="183.3795.24" TOOLBOX_PLUGIN_VERSION="0.4.6" ANNOTATION_PLUGIN_VERSION="5.3"
+  - PHPSTORM_ENV="skip incomplete" IDEA_VERSION="IU-191.6707.31-EAP-SNAPSHOT" PHP_PLUGIN_VERSION="191.6707.42" TWIG_PLUGIN_VERSION="191.6183.95" TOOLBOX_PLUGIN_VERSION="0.4.6" ANNOTATION_PLUGIN_VERSION="5.3"
+  - PHPSTORM_ENV="skip incomplete" IDEA_VERSION="IU-2019.1" PHP_PLUGIN_VERSION="191.6183.95" TWIG_PLUGIN_VERSION="191.6183.95" TOOLBOX_PLUGIN_VERSION="0.4.6" ANNOTATION_PLUGIN_VERSION="5.3"
```
Comment (cedricziel) [inline .travis.yml]:
```suggestion
  - PHPSTORM_ENV="skip incomplete" IDEA_VERSION="IU-2019.1.2" PHP_PLUGIN_VERSION="191.6707.66" TWIG_PLUGIN_VERSION="191.6183.95" TOOLBOX_PLUGIN_VERSION="0.4.6" ANNOTATION_PLUGIN_VERSION="5.3"
```

File: gradle.properties [hunk 3]
```diff
@@ -1,29 +1,25 @@
...
+phpPluginVersion = 191.6183.95
```
Comment (cedricziel) [inline gradle.properties]:
```suggestion
phpPluginVersion = 191.6707.66
```

File: gradle.properties [hunk 4]
```diff
@@ -1,29 +1,25 @@
...
+twigPluginVersion = 191.6183.95
```
Comment (cedricziel) [inline gradle.properties]:
```suggestion
twigPluginVersion = 191.6183.95
```

**Pull request merged**